api/swagger.yaml

```diff
@@ -243,6 +243,9 @@ definitions:
       ReadOnly:
         description: "Whether the mount should be read-only."
         type: "boolean"
+      Consistency:
+        description: "The consistency requirement for the mount: `default`, `consistent`, `cached`, or `delegated`."
```

Comment:
`default` is `consistent` except with the lowest (instead of highest) priority in cases where multiple consistency requirements overlap. In the future, `default` may be changed to be `cached` except with the lowest (instead of the second highest) priority in cases where multiple consistency requirements overlap.